css/css-backgrounds/border-image-shorthand-003.htm

```diff
@@ -1,16 +1,22 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml">
-    <head>
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
         <title>CSS Test: Omitted values for 'border-image-width' properties in 'border-image' shorthand</title>
-        <link rel="author" title="Microsoft" href="http://www.microsoft.com/" />
-        <link rel="help" href="http://www.w3.org/TR/css3-background/#the-border-image-shorthand" />
-        <meta name="flags" content="image" />
-        <meta name="assert" content="This test checks that the omitted 'border-image-width' value in shorthand property is set to the initial value." />
-        <style type="text/css">
+        <link rel="author" title="Microsoft" href="http://www.microsoft.com/">
+        <!--
+        Test rehabilitated by Gérard Talbot
+        2020-05-27
+        -->
+        <link rel="help" href="https://www.w3.org/TR/css-backgrounds-3/#the-border-image">
+        <link rel="match" href="reference/border-image-shorthand-003-ref.html">
+        <meta name="flags" content="">
+        <meta name="assert" content="This test checks that the omitted 'border-image-width' value in shorthand property is set to the initial value.">
+        <style>
             #test
             {
                 border: 20px double red;
```

Comment:
This test needs a border-image-width declaration here that will be overridden by the shorthand below.

Reply:
There is a border-image-width declaration at line 31.

Reply:
Maybe you meant to say that a border-image-outset declaration was missing in the longhand version.

Reply:
Yes, but that does not apply to #test.

Reply:
I am confused. border-image-shorthand-003.htm is about testing the omission of border-image-width in the shorthand form. border-image-shorthand-003.htm does that. And div#reference declares the longhand form of border-image-width.

I just looked again at the test (with the Raw button) and I see nothing to correct or to improve.